docs/application/dotnet/tutorials/certificates/managing-certificate-profile.md

```diff
@@ -24,7 +24,7 @@ However, author certificates rarely needs to be changed.
 
 This section explains how to change distributor certificate.
 
-Follow the same steps to create a certificate profile as described in `4. Create new or select old profile` of [Creating Certificate Profile](creating-certificates.md#creating-certificate-profile)
+Follow the same steps to create a certificate profile as described in [4. Create new or select old profile](creating-certificates.md#create_new_or_select_old_profile)
```

Comment:
Suggestion:
Follow the same steps to create a certificate profile as described in [create new or select old profile](creating-certificates.md#create_new_or_select_old_profile).
Please verify.

Reply:
done